docs/examples/index.rst

```diff
@@ -1,41 +1,74 @@
 Examples
 ========
 
-The examples below highlight key elements of how ownership and control information is built up through the use of statements.
+These fictional examples highlight key elements of how ownership and control information is built up through the use of statements.
 
-You can also find examples of valid BODS JSON files in the standard repository:
+A single direct owner
+---------------------
 
-- Example data (`v0.3 <https://github.com/openownership/data-standard/tree/0.3.0/examples>`__  |  `v0.2 <https://github.com/openownership/data-standard/tree/0.2.0/examples>`__  | `v0.1 <https://github.com/openownership/data-standard/tree/0.1.0/examples>`__ )
-- Test data (`v0.3 <https://github.com/openownership/data-standard/tree/0.3.0/tests/data/entity-statement/valid>`__  |  `v0.2 <https://github.com/openownership/data-standard/tree/0.2.0/tests/data/entity-statement/valid>`__  | `v0.1 <https://github.com/openownership/data-standard/tree/0.1.0/tests/data/entity-statement/valid>`__ )
+Three statements (Entity, Person and Relationship) that describe the 100% beneficial ownership of Profitech Ltd by Jennifer Hewitson-Smith. 
 
-For a visual rendering of these examples, use the `BODS Visualiser <https://www.openownership.org/visualisation/visualisation-tool/>`_.
+.. literalinclude:: ../../examples/bods-package.json
+    :language: json
 
+Tecido
+------
 
-A single direct owner
----------------------
+This example demonstrates a gradual transfer of ownership from a single owner to an employee owned trust. In this jurisdiction a beneficial owner must be declared when they own at least 25% of the company. 
 
-The example below presents three statements (Entity, Person and OwnershipOrControl) that describe the 100% beneficial ownership of Chrinon Ltd by Chris Taggart. 
+The trust itself is not represented here, shares are shown as being owned by a professional holding company. This could be the case where a jurisdiction has no trust register or a separate trust register. 
 
--0.3 JSON EXAMPLE REMOVED SO DOCS BUILD-
+`Example JSON <https://github.com/openownership/data-standard/tree/0.4.0/examples/tecido.json>`__ 
 
-Joint ownership
----------------
+| **20th January 2019** 
```

Comment:
I think, for this example, it's good to follow the dates with the declaration reference, so:

**20th January 2019. Declaration bo-jtc-8755982746**

...or similar

And what about subtitling this section 'Timeline' and doing the same for the Fermcat example?

Is the '\|' meant to be formatting the lines somehow... it doesn't seem to do anything on my docs build.

Reply:
Yeah the | makes a new line without making a new paragraph. Looks a bit neater here